src/native-bindings.js

```diff
@@ -61,7 +61,7 @@ bindings.nativeGl = (nativeGl => {
   }
   WebGLRenderingContext.onconstruct = null;
   return WebGLRenderingContext;
-})(bindings.nativeGl);
+})(bindings._nativeGl);
```

Comment:
I don't quite understand the need for this. Was this code running twice in one execution or something?

Reply:
bindings.js only gets required once and kept throughout test executions. But all the other modules will re-run, causing bindings.js to get overwritten and rewrapped.

There's a comment in bindings.js: `Due to the way `require` works with multithreading (used for `Worker`), we use this route to make sure the binaries only get initialized once. This might change with node's native "worker" support that's coming.`

Reply:
> the other modules will re-run

Which other modules is this referring to?

That comment probably doesn't apply to `bindings.js`, but `setNativeBindingsModule`.

#### Overview as I see it

- `setNativeBindingsModule` is called on bootstrap
- `setNativeBindingsModule` requires `native-bindings.js` which requires the node binary once, and then wraps it in JS
- `setNativeBindingsModule` attaches the result of this to `bindings.js` for global exposure, but this is a temporary hack in file renaming
 - the above should happen once on startup, and bindings should never be touched again, either reassigned or changed; if they are that seems like possibly a bug

Reply:
Would the [tests call](https://github.com/webmixedreality/exokit/blob/master/tests/unit/__setup.test.js#L10) be causing problems?

I think if you require `exokit`'s main, you'll run `setNativeBindingsModule`. But tests are running that as well.

Reply:
I removed the call from setNativeBindingsModule which is harmless, but if I revert the change that prevents duplicate wrapping of nativeGl, then the test still fails. 

Perhaps it's still better to make native-bindings.js more robust in case it gets invoked more than once, else it's pretty fragile. 

> the above should happen once on startup, and bindings should never be touched again, either reassigned or changed; if they are that seems like possibly a bug

I guess it happens again due to the way the test runner works in watch mode, re-requiring everything and re-initializing *all* the modules in the codebase. I'm not 100% sure what's happening under the hood, if you want to poke around. If I updated the tests in watch mode 10 times, the nativeGl wrap would get wrapped 10 times; I just assumed `bindings.js` stayed static while `native-bindings.js` got re-initialized multiple times.

PR is ready otherwise though.

Reply:
> I guess it happens again due to the way the test runner works in watch mode, re-requiring everything and re-initializing all the modules in the codebase.

I see, my hunch is that's probably not going to work, especially once `Worker` gets involved in other threads.

Basically if the `_nativeGl` my theory is it's covering up a much more serious problem, such as `mocha` watch not being compatible with the threading architecture.

I'll take the PR for a spin.

Reply:
I ended up solving this without and rename or chance of re-wrapping by making sure to create a fresh object on re-require: https://github.com/webmixedreality/exokit/pull/499/commits/0211d47e7a67c655c2e26ada0944edba5a6094e3